modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -48,7 +80,21 @@ $ oc patch Network.operator.openshift.io cluster --type='merge' \
 This step does not deploy OVN-Kubernetes immediately. Instead, specifying the `migration` field triggers the Machine Config Operator (MCO) to apply new machine configs to all the nodes in the cluster in preparation for the OVN-Kubernetes deployment.
 ====
 
-. Optional: You can disable automatic migration of several OpenShift SDN capabilities to the OVN-Kubernetes equivalents:
+.. Check that the reboot is finished by running the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
+
+.. Check that all cluster Operators are available by running the following command:
++
+[source,terminal]
+----
+$ oc get co
+----
+
+.. Alternatively: You can disable automatic migration of several OpenShift SDN capabilities to the OVN-Kubernetes equivalents:
```

Comment:
Is this an optional step or an alternative step? If it's an alternative step, is it an alternative to steps a and b? If it's an optional step, replace "Alternatively" with "Optional", per IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#indicating-optional-and-conditional-steps

Reply:
Tricky question. It was an optional step that QE and SME have said is an `Alternative` step. That's actually a big part of this bug 😄